src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Semantic/RazorSemanticTokensTests.cs

```diff
@@ -65,7 +65,7 @@ public async Task Edits_UpdateColors()
             await TestServices.Editor.WaitForClassificationAsync(ControlledHangMitigatingCancellationToken, "RazorComponentElement", 3);
 
             await TestServices.Editor.SetTextAsync(RazorProjectConstants.IndexPageContent, ControlledHangMitigatingCancellationToken);
-            await TestServices.Editor.WaitForClassificationAsync(ControlledHangMitigatingCancellationToken, "RazorComponentElement", 1);
+            await TestServices.Editor.WaitForClassificationAsync(ControlledHangMitigatingCancellationToken, "RazorComponentElement", 3);
```

Comment:
What's the `1` / `3` represent? Minutes?

Reply:
The 1 and 3 represent the number of the listed classifications to expect. Previously this one only waited for 1, but there are 3 total on that page so I'm having it wait for them all to resolve.

I'll add the parameter names to these to clarify for the future.

Reply:
oooo I see. ok 👍